### PR TITLE
Telegram refactor chat handler Now with /stardust command

### PR DESCRIPTION
**Pull request by javajohnHub**
I refactored chat_handler and telegram_handler

Everything works. its clean. It is bug free. It is ready for merge.

added new command 
/stardust 

returns total current stardust.

I cleaned the code up and separated the data from telegram so chat_handler handles all of those duties.

### CODE REVIEW

**Comment by javajohnHub** [comment]
Someone please review and approve this refactor.

File: pokemongo_bot/cell_workers/recycle_items.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def _schedule_next_force(self):
         self._next_force = self._get_next_force_schedule()
         self.emit_event(
             'next_force_recycle',
-            formatted="Next forced item recycle at {time}",
+            formatted="Next forced item recycle at {}".format(str(self._next_force.strftime("%H:%M:%S"))),
```
Comment (YvesHenri) [inline pokemongo_bot/cell_workers/recycle_items.py]: ???

Comment (javajohnHub) [inline pokemongo_bot/cell_workers/recycle_items.py]: During refactor I was getting an error with force_item_recycle. By changing the formatting the error went away.

**Comment by solderzzc** [comment]
👍

**Pull request merged**